index.html

```diff
@@ -1,271 +1,325 @@
+      <div class="about-us-border"></div>
+      <div class="ellipse1"></div>
+      <div class="about-skill">
+        <ul class="about-skill-item">
+          <li class="about-skill-items">
+            <p class="skill-language">
+              <i
+                ><img src="assets/images/Rectangle2.svg" alt="" width="17"
+              /></i>
+              <strong class="language">Language</strong>
+            </p>
+            <ul class="margin-top-10">
+              <li class="language-items">Javascript</li>
+              <li class="language-items">Ruby</li>
+              <li class="language-items">Html</li>
+              <li class="language-items">Css</li>
+            </ul>
+          </li>
+          <li class="about-skill-items">
+            <p class="skill-language">
+              <i
+                ><img src="assets/images/Rectangle3.svg" alt="" width="17"
+              /></i>
+              <strong class="language margin-05">Frameworks</strong>
+            </p>
+            <ul class="margin-top-10">
+              <li class="language-items">Bootstrap</li>
+              <li class="language-items">Ruby on Rails</li>
+              <li class="language-items">RSPec</li>
+              <li class="language-items">CapyBara</li>
+              <li class="language-items">Selenium</li>
+            </ul>
+          </li>
+          <li class="about-skill-items">
+            <p class="skill-language">
+              <i
+                ><img src="assets/images/Rectangle1.svg" alt="" width="17"
+              /></i>
+              <strong class="language margin-05">Skills</strong>
+            </p>
+            <ul class="margin-top-10">
+              <li class="language-items">Codekit</li>
+              <li class="language-items">Github</li>
+              <li class="language-items">Codepen</li>
+              <li class="language-items">Gitlab</li>
+              <li class="language-items">Terminal</li>
+            </ul>
+          </li>
+        </ul>
       </div>
-    </article>
-    <article class="card-work">
-      <div class="cards card-background-image">
-        <div class="card-name">
-          <h2 class="card-headline">Profesional Art Printing Data</h2>
+    </section>
+    <!-- contact-us -->
+    <footer id="contact-us">
+     <div class="contact-us">
+        <div class="contact-us-text">
+          <p class="contact-us-paragraph">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
         </div>
-        <div class="card-text">
-          <p class="card-headline-text">A daily selection of privately personalized reads; no accounts or sign-ups
-            required. has been the industry's standard</p>
+       <form action="submit" method="post" id="form" class="form">
+        <div class="form-control">
+          <input type="text" placeholder="Full name" id="username" />
+          <div class="input-line"></div>
         </div>
```

Comment:
- [ ] It would be nice to include a `maxlength="30"` attribute for the **_name_** form control as specified in the project requirements. Implementing this would ensure that the number of characters inputted in the name field would not exceed 30. 😄